playwright/_impl/_py37ThreadedChildWatcher.py

```diff
@@ -0,0 +1,377 @@
+# This is the LICENSE from the Python project:
```

Comment:
This does not look like a compatible permissive license we would be able to use.